src/edu/asu/plp/tool/prototype/model/PLPProject.java

```diff
@@ -145,8 +147,13 @@ public PLPProject(String name)
 	@Override
 	public void save()
 	{
-		// TODO: implement
-		throw new UnsupportedOperationException("Not Yet Implemented");
+		File file = new File(this.getPath());
+		file.mkdirs();
+		System.out.println(this.getPath());
+		System.out.println(this.getName());
+		//System.out.println(this.getFileCount());
+
+		//throw new UnsupportedOperationException("Not Yet Implemented");
```

Comment:
In this case the method isn't finished being implemented, so the TODO and UnsupportedOperationException should stay.